tools/functional-tester/etcd-tester/tester.go

```diff
@@ -80,45 +79,40 @@ func (tt *tester) runLoop() {
 		}
 	}
 
-	plog.Printf("%s functional-tester is finished", tt.logPrefix())
+	plog.Infof("%s functional-tester is finished", tt.logPrefix())
 }
 
-func (tt *tester) doRound(round int) (bool, error) {
+func (tt *tester) doRound(round int) error {
 	for j, f := range tt.failures {
 		caseTotalCounter.WithLabelValues(f.Desc()).Inc()
 		tt.status.setCase(j)
 
 		if err := tt.cluster.WaitHealth(); err != nil {
-			plog.Printf("%s wait full health error: %v", tt.logPrefix(), err)
-			return false, nil
+			return fmt.Errorf("wait full health error: %v", err)
 		}
-		plog.Printf("%s injecting failure %q", tt.logPrefix(), f.Desc())
+		plog.Infof("%s injecting failure %q", tt.logPrefix(), f.Desc())
 		if err := f.Inject(tt.cluster, round); err != nil {
-			plog.Printf("%s injection error: %v", tt.logPrefix(), err)
-			return false, nil
+			return fmt.Errorf("injection error: %v", err)
```

Comment:
Previous, we were able to keep running tests while failure happens in `Inject`. Now one failure would stop the whole tester, right?

@xiang90 Do you remember why we allowed the tester to keep running with failures?